scripts/bootstrap/config.js

```diff
@@ -0,0 +1,7 @@
+var config = {};
```

Comment:
I'll leave it as is for now, since it's accessed via the dot format that it's defined.